README.md

```diff
@@ -52,10 +52,11 @@ prompt([options, parentBrowserWindow]).then(...).catch(...)
 | ------------- | ------------- |
 | title  | (optional, string) The title of the prompt window. Defaults to 'Prompt'. |
 | label  | (optional, string) The label which appears on the prompt for the input field. Defaults to 'Please input a value:'. |
+| buttonLabels | (optional, object) The text for the OK/cancel buttons. Properties are 'ok' and 'cancel' and override the defaults of 'OK' and 'Cancel'. |
```

Comment:
The text for the OK/Cancel buttons. Properties are 'ok' and 'cancel', and defaults to null.

Reply:
Done (although I used a separate sentence for 'Defaults to null' which seemed to match the rest of the entries more..